CppParser/include/Poco/CppParser/CppParser.h

```diff
@@ -31,7 +31,7 @@
 // CppParser_API functions as being imported from a DLL, whereas this DLL sees symbols
 // defined with this macro as being exported.
 //
-#if (defined(_WIN32) || defined(__CYGWIN__)) && defined(POCO_DLL)
+#if (defined(_MSC_VER) || defined(__CYGWIN__)) && defined(POCO_DLL)
```

Comment:
@ark0f these should probably be properly checked for MSVC compiler, see #2428 why and how it was fixed in https://github.com/pocoproject/poco/commit/5de2bf7ecffa35b1ecdc07a147942b0c6ff9162a

Reply:
I understood. It will be fixed